jf_agent/git/bitbucket_server.py

```diff
@@ -248,13 +252,17 @@ def get_default_branch_commits(
                 default_branch = (
                     api_repo.default_branch['displayId'] if api_repo.default_branch else ''
                 )
+                if verbose:
+                    print("Getting list of commits.")
```

Comment:
I would maybe add the associated repo here: `Getting list of commits for {repo['name']}`